tools/extras/install_portaudio.sh

```diff
@@ -82,7 +82,7 @@ if [ -z "$MACOS" ]; then
 fi
 
 ./configure --prefix=`pwd`/install --with-pic
-sed -i.bk '40s:src/common/pa_ringbuffer.o::g; 40s:$: src/common/pa_ringbuffer.o:' Makefile
+perl -i.bk -pe '40s:src/common/pa_ringbuffer.o::g; 40s:$: src/common/pa_ringbuffer.o:' Makefile
```

Comment:
This didn't seem to work for me, perl said syntax error.
Sed and perl syntax can be quite different.

Reply:
Yes, thanks for pointing this out. I am making this and other modifications now. Some of the other edits such as deleting matching patterns are easier using `grep -v` than with `perl`.